src/jokesData.js

```diff
@@ -0,0 +1,33 @@
+const jokesData = [
```

Comment:
Looks like you do not need jokes any more. Please delete it if so.

Reply:
File deleted